podbuilder/podbuilder.go

```diff
@@ -152,7 +152,10 @@ func (pb *PodBuilder) PodForClient(client *grpcv1.Client) (*corev1.Pod, error) {
 
```

Comment:
Is it possible that we add "timeout --kill-after=$KILL_AFTER $POD_TIMEOUT"  at line 189 when we create pod instead in each yaml file?

Reply:
It is a great point, we would have been okay if we have run command for all languages exposed directly, but we have php7, which has to be wrapped with other reason. We might could have "timeout --kill-after=$KILL_AFTER $POD_TIMEOUT"  as a prefix, but for php7 it would be timeout the run_php7_worker.sh instead of the worker process itself.  For php7 we actually could add the timeout inside the wrapper , but that way php7 would have double timeout, not consistent with the rest of the languages. 

Generally, I think adding ENV in podbuilder is much safer since you can abandon it with no visible consequences, but if you add argument or command (or modified the command in anyway), it will always trying to execute it, if in the future we have to wrap more workers, this could be problematic.

@codeblooded @paulosjca

Reply:
I prefer environment variables as mentioned by @wanlin31. Concatenating the strings in podbuilder seems fragile to me. It also may break or limit some commands.

Reply:
And it probably need more work than I thought ( tried could not make it work easily...